 
-B is a remnant from system V m4, not implemented in our m4, and a noop in gm4

--- crypto/des/Makefile.orig	Thu Jul  9 14:02:56 2015
+++ crypto/des/Makefile	Sat Sep 12 20:40:03 2015
@@ -60,7 +60,7 @@ des: des.o cbc3_enc.o lib
 	$(CC) $(CFLAGS) -o des des.o cbc3_enc.o $(LIB)
 
 des_enc-sparc.S:	asm/des_enc.m4
-	m4 -B 8192 asm/des_enc.m4 > des_enc-sparc.S
+	m4 asm/des_enc.m4 > des_enc-sparc.S
 dest4-sparcv9.s:	asm/dest4-sparcv9.pl
 	$(PERL) asm/dest4-sparcv9.pl $(CFLAGS) > $@
 
